### PR TITLE
fixed responsiveness on mobile

### PR DESCRIPTION
**Pull request by JTravaux**
Got a ticket from someone on a mobile device with a responsiveness issue. I modified the css classes on the modal a bit so it should work better on more devices.

### CODE REVIEW

File: src/app/auction-page/auction-page.component.html [hunk 1]
```diff
@@ -699,12 +696,13 @@
...
           step="1"
           type="number"
-          class="axn-input axn-input-mid"
+          class="axn-input"
           autocomplete="off"
           required="required"
           style="width: auto;"
           name="stake-days"
           [max]="stakeMaxDays"
+          style="font-size: 2rem;"
```
Comment (Mvii7) [inline src/app/auction-page/auction-page.component.html:705]: Styles need to go to sass file

Comment (JTravaux) [reply src/app/auction-page/auction-page.component.html]: Just removed it, wasn't making too much of a visual difference anyway.

File: src/app/auction-page/auction-page.component.html [hunk 2]
```diff
@@ -685,10 +685,7 @@
   <div class="dialog-block__content">
     <div class="dialog-title">Stake Auction Bid</div>
 
-  
-    <div class="dialog-description">
-
-    
+    <div class="dialog-description" style="padding: 10px;">
```
Comment (Mvii7) [inline src/app/auction-page/auction-page.component.html:688]: This one as well

File: src/app/auction-page/auction-page.component.html [hunk 3]
```diff
@@ -699,7 +696,7 @@
         <input
           step="1"
           type="number"
-          class="axn-input axn-input-mid"
+          class="axn-input"
           autocomplete="off"
           required="required"
           style="width: auto;"
```
Comment (Mvii7) [inline src/app/auction-page/auction-page.component.html:702]: This one as well while we're at it

**Pull request merged**